examples/00_getting_started.py

```diff
@@ -58,10 +58,7 @@
         pd_init = {'resource'      : resource,
                    'runtime'       : 30,  # pilot runtime (min)
                    'exit_on_error' : True,
-                   'project'       : config[resource]['project'],
-                   'queue'         : config[resource]['queue'],
-                   'access_schema' : config[resource]['schema'],
-                   'cores'         : config[resource]['cores'],
+                   'cores'         : 4 * 41 * 14,
```

Comment:
Is this intended? I don't think so as we used the config file so to avoid resource-specific default configs?

Reply:
Thanks, reverted.